.esdoc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Also check out https://esdoc.org and let me know what you think—it autogenerates from ES6 code which is nice (unlike jsdoc, where you have to add annotations), but if you don't think we need that we could use something a little more stable.